config/locales/en.yml

```diff
@@ -1134,7 +1134,7 @@ en:
         both: "Both SMS and email"
     recipient_selection_options:
       all_users: "All users in mission"
-      all_observers: "All observers in mission"
+      all_enumerators: "All enumerators in mission"
```

Comment:
I think key changes will need to be made in all the translation files, right?

Reply:
I wasn't sure about this - how will translators for other languages know those ones should be updated?

Reply:
They will appear as new strings that need to be translated I believe. (If the key changes).

If the key doesn't change, not sure. Presumably Transifex should be smart enough to see there was a change to the value and prompt the translator to doublecheck?

Reply:
updated keys in all the files